creator-node/src/app.js

```diff
@@ -63,6 +37,48 @@ app.use(errorHandler)
  * @param {ServiceRegistry} serviceRegistry object housing all Content Node Services
  */
 const initializeApp = (port, serviceRegistry) => {
```

Comment:
all above code is just moved into here with no changes
had to do this so that promBundle could be applied before routes are applied